html/elements/meta.json

```diff
@@ -124,6 +124,469 @@
               "standard_track": true,
               "deprecated": false
             }
+          },
+          "height": {
+            "__compat": {
+              "description": "content=\"height=[value]\"",
+              "mdn_url": "https://developer.mozilla.org/docs/Web/HTML/Viewport_meta_tag#height",
+              "spec_url": "https://drafts.csswg.org/css-viewport/#viewport-meta",
+              "support": {
+                "chrome": {
+                  "version_added": false
+                },
+                "chrome_android": {
+                  "version_added": "18"
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": {
+                  "version_added": "4"
+                },
+                "ie": {
+                  "version_added": false
+                },
+                "oculus": "mirror",
+                "opera": "mirror",
+                "opera_android": {
+                  "version_added": "10.1"
+                },
```

Comment:
The [Opera Mobile 10.1 Beta release notes](https://web.archive.org/web/20140919184929/https://dev.opera.com/blog/opera-mobile-10-1-beta-for-android-is-here/) mention `viewport`, but don't seem to mention `height`.